rewrite-weblogic/rewrite.yml

```diff
@@ -0,0 +1,6 @@
+type: specs.openrewrite.org/v1beta/recipe
+name: com.oracle.weblogic.RecipeBestPractices
```

Comment:
In IntelliJ IDEA Ultimate 2024.1+ this becomes a runnable recipe with a single click:
![image](https://github.com/user-attachments/assets/84a75370-f26d-437a-8e82-a8ae3f46c6b2)